internal/jimm/jimm.go

```diff
@@ -42,74 +36,11 @@ import (
 	"github.com/canonical/jimm/v3/internal/openfga"
 	ofganames "github.com/canonical/jimm/v3/internal/openfga/names"
 	"github.com/canonical/jimm/v3/internal/pubsub"
+	"github.com/canonical/jimm/v3/pkg/api/params"
 	apiparams "github.com/canonical/jimm/v3/pkg/api/params"
 	jimmnames "github.com/canonical/jimm/v3/pkg/names"
 )
 
-var (
-	initiateMigration = func(ctx context.Context, j *JIMM, user *openfga.User, spec jujuparams.MigrationSpec) (jujuparams.InitiateMigrationResult, error) {
-		return j.InitiateMigration(ctx, user, spec)
-	}
-)
-
-// OAuthAuthenticator is responsible for handling authentication
-// via OAuth2.0 AND JWT access tokens to JIMM.
-type OAuthAuthenticator interface {
```

Comment:
This interface has been removed because I realised it was duplicating the same interface in the `internal/jimm/login` package.

Reply:
hmm.. using an interface defined in another package is a bit dirty.. though i will concede that godocs on this interface went a bit deep into implementation..

Reply:
Yeah, I'm c/p my message from Matrix,
>the internal/jimm/juju package is the one requiring the dialer, the API, etc. And internal/jimm/jimm.go is just the constructor of the manager. So internal/jimm/jimm.go doesn't use the interfaces, but it needs them for the constructor signature.